crux-test/test/crux/tpch_test.clj

```diff
@@ -0,0 +1,462 @@
+(ns crux.tpch-test
+  (:require [clojure.test :as t]
+            [crux.fixtures :as f]
+            [crux.fixtures.api :as fapi :refer [*api*]]
+            [crux.fixtures.calcite :as cf]
+            [crux.fixtures.kv :as kvf]
+            [crux.fixtures.standalone :as fs]
+            [crux.fixtures.tpch :as tf])
+  (:import io.airlift.tpch.TpchTable))
+
+;; Transaction Processing Performance Council
+;; http://www.tpc.org/
+
+(defn- with-tpch-schema [f]
+  (f/transact! *api* (tf/tpch-tables->crux-sql-schemas))
+  (f))
+
+(defn- with-tpch-dataset [f]
+  (doseq [^TpchTable t (TpchTable/getTables)]
+    ;; Using a scale-factor of .005 and no take, takes ~3 mins combined for all tests
+    (f/transact! *api* (take 10 (tf/tpch-table->docs t))))
```

Comment:
I wonder if this test belongs in crux-bench? or maybe we put the full test in crux-bench and leave a subset here

Reply:
Currently all tpch tests runs with a very small amount of docs (5 per tpch type). We could up the type and perhaps move to bench, but I judge we're OK for this PR as is? I've found the tpch tests are quick enough.

Reply:
We'll revisit taking advantage of TCPH in future.